admin_manual/release_notes.rst

```diff
@@ -441,12 +443,10 @@ Protecting ownCloud on IIS from Data Loss
 Under certain circumstances, running your ownCloud server on IIS could be at 
 risk of data loss. To prevent this, follow these steps.
 
-In your ownCloud server configuration file, ``owncloud\config\config.php``, set 
+* In your ownCloud server configuration file, ``owncloud\config\config.php``, set 
```

Comment:
Just trying to fix the broken https://doc.owncloud.org/server/9.2/admin_manual/release_notes.html#protecting-owncloud-on-iis-from-data-loss here